solarwinds_apm/distro.py

```diff
@@ -65,10 +71,42 @@ def _log_runtime(self):
         self._log_python_runtime()
         logger.info("OpenTelemetry %s/%s", sdk_version, inst_version)
 
+    def _get_token_from_service_key(self):
+        """Return token portion of service_key if set, else None"""
+        service_key = environ.get("SW_APM_SERVICE_KEY")
+        if not service_key:
+            logger.debug("Missing service key")
+            return None
+        # Key must be at least one char + ":" + at least one other char
+        key_parts = [p for p in service_key.split(":") if len(p) > 0]
+        if len(key_parts) != 2:
+            logger.debug("Incorrect service key format")
+            return None
+        return key_parts[0]
+
     def _configure(self, **kwargs):
         """Configure default OTel exporter and propagators"""
         self._log_runtime()
 
+        # Set defaults for OTLP logs export by HTTP to SWO
+        header_token = self._get_token_from_service_key()
+        if not header_token:
+            logger.debug(
+                "Setting OTLP logging defaults without valid SWO token"
+            )
+        environ.setdefault(
```

Comment:
These 4 env vars always have defaults set. They will be overridden by user-set env vars. They will not be used by Configurator is export_logs is not enabled.